src/app/api/prepare-migration/prepare-migration.ts

```
@@ -1,9 +1,19 @@
-import mysql2 from 'mysql2/promise'
-import fs from 'fs'
-import path from 'path'
+import * as mysql2 from 'mysql2/promise'
+import * as fs from 'fs'
+import * as path from 'path'
 import { RowDataPacket } from 'mysql2'
-import { WordPressPost, SanityContent, MigrationRecord } from '@/types/migration'
-import { extractMediaFromContent, mapMediaToLocalPaths, replaceMediaUrls, generateMediaStats } from '@/utils/media-processor'
+import type {
+  WordPressPost,
+  SanityContent,
```

ESLint

[error] 'SanityContent' is defined but never used.

(@typescript-eslint/no-unused-vars)